src/components/common/StarRating/StarRating.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import styles from './StarRating.modules.scss';
+import PropTypes from 'prop-types';
+
+const addMyRatingToState = (id, addRating, value, myRating, event) => {
+  event.preventDefault();
+  addRating({ id: id, myRating: value });
+};
+
+const colorSet = (rating, value, hoverRating, stars) => {
+  if (hoverRating >= value) {
+    return '#e6e29c';
+  } else if (!hoverRating && rating >= value) {
+    return '#e6e29c';
+  } else if (!rating && stars >= value) {
+    return '#000000';
+  }
+  return '#F2F2F2';
+};
+
+const starsArray = [1, 2, 3, 4, 5];
+
+const StarRating = ({ myRating, stars, addRating, id }) => {
+  const [hoverRating, setHoverRating] = React.useState(0);
+  const setHover = star => {
```

Comment:
A po co jest ta funkcja ? Dlaczego po prostu nie skorzystać z `setHoverRating`?

Reply:
Racja, zmieniłem to :)